shell/platform/windows/fixtures/main.dart

```diff
@@ -315,9 +315,49 @@ void drawHelloWorld() {
       ..addPicture(ui.Offset.zero, picture)
       ..pop();
 
-    ui.window.render(sceneBuilder.build());
+    ui.PlatformDispatcher.instance.implicitView?.render(sceneBuilder.build());
```

Comment:
The `window` global is deprecated and replaced by `PlatformDispatcher.instance.implicitView`. This isn't necessary for this PR, but removes a usage of a deprecated member :)